testing/jitterentropy-rngd/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Henrik Riomar <henrik.riomar@gmail.com>
+# Maintainer:
+pkgname=jitterentropy-rngd
+pkgver=1.0.8
+pkgrel=0
+pkgdesc="CPU Jitter RNG Daemon"
+url="http://www.chronox.de/jent.html"
+arch="all"
+license="GPL-2.0 BSD"
+makedepends="linux-headers"
+options="!check" # no test suite
+subpackages="$pkgname-doc $pkgname-openrc"
+source="jitterentropy-rngd-$pkgver.tar.gz::https://github.com/smuellerDD/jitterentropy-rngd/archive/v$pkgver.tar.gz
+	$pkgname.initd
+	"
+builddir="$srcdir/jitterentropy-rngd-$pkgver"
+
+build() {
+	cd "$builddir"
```

Comment:
`cd "$builddir"` is obsolete nowadays and can be removed (as abuild does that automatically).

Reply:
OK, any news on when `newapkbuild` will stop adding it?

Reply:
I submitted a PR to remove it from the template.

Reply:
thanks, will update this PR.